Cargo.toml

```diff
@@ -1,12 +0,0 @@
-[workspace]
```

Comment:
why don't we use workspace anymore?

Reply:
With the workspace I was not able to make the `ink-wrapper-types` to work with different deps' versions from `drink!` and `aleph-client`.

Reply:
okay, let me create an issue for this - I think it's doable; I'll take a look in free time; resolving for now

Reply:
It'll be possible to do it now, now that we don't have `aleph_clien` here anymore.